KachnaOnline.Dto/Events/BaseEventDto.cs

```diff
@@ -0,0 +1,86 @@
+// BaseEventDto.cs
+// Author: David Chocholatý
+
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace KachnaOnline.Dto.Events
+{
+    /// <summary>
+    /// Represents an event modification.
+    /// </summary>
+    public class BaseEventDto
+    {
+        /// <summary>
+        /// A name of the event.
+        /// </summary>
+        /// <example>Start@FIT</example>
+        [StringLength(128)]
+        [Required(AllowEmptyStrings = false)]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// A place associated with the event.
+        /// </summary>
+        /// <example>VUT BUT</example>
```

Comment:
Nitpick, perhaps `FIT BUT` would be more fitting (no pun intended)?

Reply:
Yes, that is a mistake. Fixed.